pkg/helm/controller.go

```diff
@@ -28,8 +28,9 @@ import (
 )
 
 var (
-	trueVal = true
-	commaRE = regexp.MustCompile(`\\*,`)
+	trueVal         = true
+	commaRE         = regexp.MustCompile(`\\*,`)
+	DefaultJobImage = "rancher/klipper-helm:v0.5.0-build20210505"
```

Comment:
Is this var referenced outside of this package?

Reply:
Yes, in both K3s and RKE2.